config/7.10.0/common.yaml

```diff
@@ -33,6 +33,7 @@ console:
               app: "[[.ApplicationName]]"
               application: "[[.ApplicationName]]"
               service: "[[.ApplicationName]]-[[.Console.Name]]"
+              cluster: "jgrp.k8s.console.[[.Console.Name]]"
```

Comment:
in order to ensure cluster name uniqueness within a namespace, the label should probably look more like this -
`"jgrp.k8s.console.[[.ApplicationName]].[[.Console.Name]]"`